Sources/TuistAutomation/Utilities/AutomationStorage.swift

```diff
@@ -0,0 +1,18 @@
+import Foundation
+import Mockable
+import XcodeGraph
+
+@Mockable
+/**
+ Store for automation-related metadata
+ */
+public protocol AutomationStoring: AnyObject {
+    /// Initial graph before automation mappers were applied
+    var initialGraph: Graph? { get set }
+}
+
```

Comment:
We can use this in the future to save tests being run in memory instead of in the file system as we're doing right now